index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+<header id="header" >
+  <img id="header-img"
+               src="https://res.cloudinary.com/crunchbase-production/image/upload/c_lpad,f_auto,q_auto:eco,dpr_1/vxatfbgm79t1j6ksshqc"
+               alt="Microverse logo">
+
+
+               <nav id="nav-bar">
+          <a class="nav-link" href="#features">Features</a>
+          <a class="nav-link" href="#how_it_works">How it works</a>
+          <a class="nav-link" href="#pricing">Pricing</a>
+      </nav>
+
+
+
+</header>
+
+
+<div class="container">
+
+  <div class="container">
+              <section id="features">
+                  <p>Features</p>
+              </section>
+              <section id="how_it_works">
+                  <p>How it works</p>
+              </section>
+              <section id="pricing">
+                  <p>Pricing</p>
+              </section>
+
+

```

Comment:
I think it would be better to use <section> to make the work appears in sections before started using <div> for creating your container.